pkg/vmimage/resources.go

```diff
@@ -138,19 +138,33 @@ func nic(subscriptionID, resourceGroup, location string) *network.Interface {
 	}
 }
 
-func vm(subscriptionID, resourceGroup, location string, sshPublicKey string) *compute.VirtualMachine {
+func vm(subscriptionID, resourceGroup, location string, sshPublicKey string, verify bool, image string) *compute.VirtualMachine {
```

Comment:
what about changing the signature to:

```suggestion
func vm(subscriptionID, resourceGroup, location, sshPublicKey, image string, verify bool) *compute.VirtualMachine {
```